README.md

```diff
@@ -35,6 +35,7 @@ sudo apt-get -y install \
     qemu-system-x86 \
     qemu-system-aarch64 \
     make
+sudo uv install pytest-reportportal
```

Comment:
Please check the [uv docs](https://docs.astral.sh/uv/) how to use it, you don't need `sudo` nor exists a command `install`, you may confuse it with `apt`?